packages/xrpl/test/integration/requests/subscribe.test.ts

```diff
@@ -8,27 +7,32 @@ import {
   SubscribeResponse,
 } from 'xrpl-local'
 import { StreamType } from 'xrpl-local/models/common'
+import type { LedgerStreamResponse } from 'xrpl-local/models/methods/subscribe'
 
 import serverUrl from '../serverUrl'
-import { setupClient, teardownClient } from '../setup'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
 import { ledgerAccept, subscribeDone, testTransaction } from '../utils'
 
 // how long before each test case times out
 const TIMEOUT = 20000
 
-// Note: This test use '.then' to avoid awaits in order to use 'done' style tests.
-// eslint-disable-next-line max-params -- Helps keep things well-typed
 async function createTxHandlerTest(
   client: Client,
   wallet: Wallet,
-  done: Mocha.Done,
   subscriptionStream: StreamType,
-): Promise<void> {
+): Promise<{ transactionPromise: Promise<void> }> {
```

Comment:
Why does this return an object instead of just the promise?

Reply:
I think it's cause that's how the code was before and it was meant to show that you don't want to await the promise immediately, but rather take the returned promise and await that. But I fixed it.

Reply:
That's a fair point, I'd be fine with reverting to how this was before.